backend/grants/admin.py

```diff
@@ -560,6 +561,31 @@ def emoji_gender(self, obj):
         }
         return emoji[gender]
 
+    @admin.display(
+        boolean=True,
+    )
+    def user_has_ticket(self, obj: Grant) -> bool:
+        if not obj.user_id:
+            return None
+
+        try:
+            return user_has_admission_ticket(
+                email=obj.user.email,
+                event_organizer=obj.conference.pretix_organizer_id,
+                event_slug=obj.conference.pretix_event_id,
+            )
+        except Exception as e:
```

Comment:
When does this happen?

Reply:
Locally I was getting this: 

```
403 Client Error: Forbidden for url: https://tickets.pycon.it/api/v1/organizers/python-italia/events/pycon12/tickets/attendee-has-ticket/
```

Probably i have something misconfigured... but can happen in theory also in prod?

Reply:
Hmm I don't think it can as prod? you should also probably update your local config to point to the right conference

Reply:
ok removed